### PR TITLE
[copy] add missing trademark to Mempool Goggles

### PR DESCRIPTION
**Pull request by nymkappa**

### CODE REVIEW

File: frontend/src/app/docs/api-docs/api-docs-data.ts [hunk 1]
```diff
@@ -9584,7 +9584,7 @@ export const faqData = [
     category: "advanced",
     showConditions: bitcoinNetworks,
     fragment: "how-do-mempool-goggles-work",
-    title: "How do Mempool Goggles work?",
+    title: "How do Mempool Goggles™ work?",
```
Comment (nymkappa) [inline frontend/src/app/docs/api-docs/api-docs-data.ts:9587]: Tried the `&trade;` but it does not work here

**Comment by hunicus** [comment]
ACK https://github.com/mempool/mempool/pull/4886/commits/ce6808b3c4533d685e767f18fc45ab3761f3a141

**Review by hunicus — APPROVED** [review]
ACK https://github.com/mempool/mempool/pull/4886/commits/ce6808b3c4533d685e767f18fc45ab3761f3a141

File: frontend/src/locale/messages.ca.xlf [hunk 2]
```diff
@@ -875,7 +875,7 @@
         <note priority="1" from="description">mining.3-months</note>
       </trans-unit>
       <trans-unit id="f26767fd3fbdc174a0f961fc538d10f80eaec6d8" datatype="html">
-        <source>Mempool Goggles: Accelerations</source>
+        <source>Mempool Goggless&trade;: Accelerations</source>
```
Comment (softsimon) [inline frontend/src/locale/messages.ca.xlf:878]: Don't change local language files

Comment (wiz) [inline frontend/src/locale/messages.ca.xlf:878]: this file shouldn't be edited, and you're adding a random typo?

**Pull request merged**